README.md

```diff
@@ -1,17 +1,23 @@
-# puppet-windows_autoupdate
+# Windows Automatic Updates module for Puppet
+
+[![Build Status](https://travis-ci.org/voxpupuli/puppet-windows_autoupdate.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-windows_autoupdate)
+[![Code Coverage](https://coveralls.io/repos/github/voxpupuli/puppet-windows_autoupdate/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-windows_autoupdate)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/windows_autoupdate.svg)](https://forge.puppetlabs.com/puppet/windows_autoupdate)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/windows_autoupdate.svg)](https://forge.puppetlabs.com/puppet/windows_autoupdate)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/windows_autoupdate.svg)](https://forge.puppetlabs.com/puppet/windows_autoupdate)
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/windows_autoupdate.svg)](https://forge.puppetlabs.com/puppet/windows_autoupdate)
 
 #### Table of Contents
 
 1. [Overview](#overview)
-2. [Module Description - What is the windows_autoupdate module?](#module-description)
-3. [Setup - The basics of getting started with windows_autoupdate](#setup)
+1. [Module Description - What is the windows_autoupdate module?](#module-description)
+1. [Setup - The basics of getting started with windows_autoupdate](#setup)
```

Comment:
Actually, scrap that.  https://github.com/voxpupuli/puppet-windows_eventlog/pull/15 was merged and the README there looks fine.